packages/react-catalog-view-extension/src/components/CatalogTile/__snapshots__/CatalogTile.test.tsx.snap

```diff
@@ -65,28 +65,30 @@ exports[`CatalogTile renders properly 1`] = `
             class="catalog-tile-pf-badge-container"
           >
             <span>
-              <span
```

Comment:
can you rebase? looks like these snapshots snuck in here

Reply:
Will do